client/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.html

```diff
@@ -0,0 +1,45 @@
+<section class="telecom-telephony-guides">
+
+    <div class="text-center" data-ng-if="$ctrl.loading.init">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div class="row"
+         data-ng-if="!$ctrl.loading.init">
+        <div class="col-xs-12 col-md-6">
+
+            <div class="alert alert-info"
+                 role="alert"
+                 data-ng-if="!$ctrl.hasGuides()"
+                 data-translate="telephony_guides_empty">
+            </div>
+
+            <section class="widget-presentation"
+                     data-ng-if="$ctrl.hasGuides()"
+                     data-ng-repeat="section in $ctrl.guides.sections track by section.label">
+                <header class="widget-presentation-header">
+                    <h2 class="widget-presentation-title"
+                        data-translate="{{ section.label }}">
+                    </h2>
+                </header>
+                <ul>
+                    <li data-ng-repeat="guide in section.guides track by guide.label">
+                        <a title="{{ guide.label | translate }} ({{:: 'common_open_new_tab' | translate }})"
+                           target="_blank"
+                           rel="noopener"
+                           data-ng-if="guide.url[$ctrl.language]"
+                           data-ng-href="{{ guide.url[$ctrl.language] }}">
+                            <span data-translate="{{ guide.label | translate }}"></span>
+                            <i class="ovh-font ovh-font-newtab" aria-hidden="true"></i>
```

Comment:
```suggestion
                             <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
                             <span class="sr-only" data-translate="common_open_new_tab" aria-hidden="true"></span>
```

Reply:
Sorry, but it's not the same style between the two versions of code